electroncash_gui/qt/token_send.py

```diff
@@ -842,7 +860,9 @@ def clear_form(self):
         self.amount_e.clear()
         if self.form_mode == self.FormMode.send:
             self.te_payto.clear()
-        self.te_desc.clear()
+        elif self.form_mode == self.FormMode.mint:
+            self.te_desc.clear()
+        self.nfts_to_mint.clear()
```

Comment:
I think you reversed the 2 lines here....?

Reply:
Gah, yes.  Sorted.